.pre-commit-config.yaml

```diff
@@ -62,4 +62,5 @@ repos:
     hooks:
       - id: mypy
         name: Check types
+        additional_dependencies: [torch==2.1.1]
```

Comment:
This was necessary to resolve the discrepancy between pre-commit env (where `torch` wasn't installed) and the local dev env (where `torch` is installed).